SUMMARY.md

```diff
@@ -13,11 +13,13 @@
   - [Analog Sensor Top](./hw/top_earlgrey/ip/ast/README.md)
   - [Alert Handler](./hw/top_earlgrey/ip_autogen/alert_handler/README.md)
     - [Design Verification](./hw/top_earlgrey/ip_autogen/alert_handler/dv/README.md)
+      - [Testplan](./hw/top_earlgrey/ip_autogen/alert_handler/data/alert_handler_testplan.hjson)
```

Comment:
I'm not sure if I am a fan of having just the one subsection in most of these Design Verification sections. It's usually nice to have at least two subsections.

Reply:
https://github.com/lowRISC/opentitan/pull/17323#discussion_r1121777941